src/fetchStats.js

```diff
@@ -1,93 +0,0 @@
-const AWS = require('aws-sdk')
-
-const awsConfig = {
-  region: 'us-east-1',
-  endpoint: 'http://dynamodb.us-east-1.amazonaws.com',
-  accessKeyId: process.env.VUE_APP_AWS_DB_ACCESS_KEY,
-  secretAccessKey: process.env.VUE_APP_AWS_DB_SECRET_KEY
-}
-AWS.config.update(awsConfig)
-
-const docClient = new AWS.DynamoDB.DocumentClient()
-/**
- * Fetches gmail data according to some range
- * Formats for frontend client.
- */
-export const rangeScan = (email, start, end) => {
-  const params = {
-    TableName: 'gmail-stats',
-    KeyConditionExpression: 'id = :id AND #t between :start AND :end',
-    ExpressionAttributeNames: {
-      '#t': 'time'
-    },
-    ExpressionAttributeValues: {
-      ':id': email,
-      ':start': start,
-      ':end': end
-    }
-  }
-  return new Promise((resolve, reject) => {
-    docClient.query(params, function (err, data) {
-      if (err) return reject('gmail-stats::rangeScan::error - ' + err)
-      if (
-        !data.Items ||
```

Comment:
I believe .query always returns an object with `Items` even if it's an empty array.